src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -1171,7 +1171,7 @@ static unsafe bool TryFormatUInt64Slow(ulong value, ReadOnlySpan<char> format, I
             }
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)] // called from only one location
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Was the comment stale, or this PR is causing it to be called in more places?

Reply:
Yes. We now handle both `byte` and `char` based versions so its at least 2 places